pandas/__init__.py

```diff
@@ -236,6 +247,29 @@ class SparseDataFrame:
     class SparseSeries:
         pass
 
+    class Datetime:
```

Comment:
Nit but maybe __Datetime so we are consistent with https://github.com/pandas-dev/pandas/pull/30386/files?